site/app/views/NavigationView.php

```diff
@@ -24,16 +24,21 @@ public function showGradeables($sections_to_list) {
         $site_url = $this->core->getConfig()->getSiteUrl();
         $return.= <<<HTML
 HTML;
-        $title_to_button_type = array("FUTURE" => "btn-default", "OPEN" => "btn-primary" , "CLOSED" => "btn-danger", "ITEMS BEING GRADED" => "btn-default", "GRADED" => 'btn-success');
+        $title_to_button_type_submission = array("FUTURE" => "btn-default", "OPEN" => "btn-primary" , "CLOSED" => "btn-danger", 
+                                                 "ITEMS BEING GRADED" => "btn-default", "GRADED" => 'btn-success');
+        $title_to_button_type_grading = array("FUTURE" => "btn-default", "OPEN" => "btn-default" , "CLOSED" => "btn-default", 
+                                                 "ITEMS BEING GRADED" => "btn-primary", "GRADED" => 'btn-danger');
         $title_to_prefix = array("FUTURE" => "OPEN DATE", "OPEN" => "SUBMIT", "CLOSED" => "CLOSED", "ITEMS BEING GRADED" => "GRADING", "GRADED" => "GRADED");
         foreach($sections_to_list as $title => $gradeable_list){
+            if(count($gradeable_list)==0) continue;
```

Comment:
I hate this paradigm of single line if statements (or just not using braces in general) as it's a bit more error-prone at best.

```
if (count($gradeable_list) == 0) {
    continue;
}
```